redirects.txt

```diff
@@ -1115,4 +1115,5 @@ docs/projects/version-control/configuring-version-control-on-a-project.md -> doc
 docs/getting-started/first-deployment/configure-environments.md -> docs/getting-started/first-deployment/legacy-guide/configure-environments.md
 docs/getting-started/first-deployment/create-and-deploy-a-release.md -> docs/getting-started/first-deployment/legacy-guide/create-and-deploy-a-release.md
 docs/getting-started/first-deployment/create-projects.md -> docs/getting-started/first-deployment/legacy-guide/create-projects.md
-docs/getting-started/first-deployment/define-the-deployment-process.md -> docs/getting-started/first-deployment/legacy-guide/define-the-deployment-process.md
\ No newline at end of file
+docs/getting-started/first-deployment/define-the-deployment-process.md -> docs/getting-started/first-deployment/legacy-guide/define-the-deployment-process.md
+docs/notifications/dynamic-workers-ubuntu-1804-deprecation.md -> docs/infrastructure/workers/dynamic-worker-pools/ubuntu-1804-end-of-life.md
```

Comment:
This is needed because in PreProd this page was initially on `docs/notifications/dynamic-workers-ubuntu-1804-deprecation.md`.

Reply:
oh so you're "committed" to that previous change :(